src/lib/components/forms/fields/basic-input-field.svelte

```diff
@@ -106,7 +106,7 @@
           class="mt-s4 self-end text-f12 text-gray-text-alt"
           class:text-error={value.length > maxLength}
         >
-          {value?.length}/{maxLength} caractères
+          {value?.length ?? 0}/{maxLength} caractères
```

Comment:
pas nécessaire, on est dans un {#if value}. Le point d'interrogation initial n'était pas nécessaire non plus.
```suggestion
          {value.length}/{maxLength} caractères
```